### PR TITLE
fix(store): no error message for denied keychain access

### PR DESCRIPTION
**Pull request by korotkovao**
[BUG - Store access is outputting errors which should be warnings](https://github.com/SAP/open-ux-tools/issues/2473#top)
#2473

### CODE REVIEW

File: packages/store/src/secure-store/keytar-store.ts [hunk 1]
```diff
@@ -53,7 +53,9 @@ export class KeytarStore implements SecureStore {
         try {
             return (await this.keytar.findCredentials(service)).reduce((result, entry) => {
                 try {
-                    result[String(entry.account)] = JSON.parse(entry.password);
+                    if (entry.account) {
```
Comment (IainSAP) [inline packages/store/src/secure-store/keytar-store.ts:56]: Could you please add a comment about when a user denies permission to access the keystore that the `entry.account` is undefined (if that is whats happening?), thx

Comment (korotkovao) [reply packages/store/src/secure-store/keytar-store.ts:57]: Thank you, added the comment.

**Review by devinea — APPROVED** [review]
Code changes are clear
changeset ✅

**Review by Klaus-Keller — APPROVED** [review]
Thanks @korotkovao!
* code change to avoid error when key is not present look good
* changeset exists
* test coverage is great
* did no manual test, just visual review

Approved from my side

**Review by IainSAP — APPROVED** [review]
thx @korotkovao 

- Comments addressed
- Tests added to cover

**Comment by korotkovao** [comment]
Thank you @devinea @Klaus-Keller @IainSAP

**Pull request merged**